src/platform/errors/errorUtils.node.ts

```diff
@@ -567,21 +567,26 @@ function isBuiltInModuleOverwritten(
     };
 }
 
-export function displayErrorsInCell(
+export function endCellAndDisplayErrorsInCell(
```

Comment:
Renamed to be more explicit about the fact that the cell execution is marked as completed.